docs/content/reference/zero-trust/scoutsuite.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Scoutsuite"
+date: 2021-03-02T16:23:06+02:00
+draft: false
+---
+
+### What is ScoutSuite?
+
+<a href="https://github.com/nccgroup/ScoutSuite" target="_blank" >Scout Suite</a> is an open-source cloud security-auditing tool.
+It queries the cloud API to gather configuration data of the cloud infrastructure. Based on configuration
```

Comment:
```suggestion
It queries the cloud API to gather configuration data from the cloud infrastructure. Based on configuration
```

Reply:
This sentence tries to say that the data gathered is about cloud infrastructure, not that it's gathered from cloud infrastructure. Fixed to `It queries the cloud API to gather configuration data about the cloud infrastructure.`